tests/functional/api/gateway_test.py

```diff
@@ -35,7 +35,8 @@ def test_minimum_viable_login(self, gateway_launch, required_params):
                         "url": TEST_SETTINGS["h_api_url_public"] + "token",
                     },
                 }
-            }
+            },
+            "data": {"assignments": Any.list()},
```

Comment:
We check this in detail in the back-end, no need to duplicate it all again here.